### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by skyleilani**
Add "Create a repository to store the ML model to deploy" section and corrected JSON config information.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,22 +55,60 @@ Switch to **Raw JSON** mode and add the following to your `"modules"` array:
...
+
 > [!NOTE]  
 > For more information, see [Configure a Robot](https://docs.viam.com/manage/configuration/).
 
+
+### Create a repository to store the ML model to deploy
+
+Currently, you must manually create a Triton [model repository](https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/user_guide/model_repository.html).
+On your robot's Jetson computer, create a [structured repository](https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/user_guide/model_repository.html) under the `~/.viam` directory.
+The exact subpath under `~/.viam` does not matter.
```
Comment (bhaney) [inline README.md:84]: I tried this today, and it didn't work unless I put it on my Orin's `/root/.viam` directory. I think it depends from which `.viam` directory the user runs from.

**Review by bhaney — CHANGES_REQUESTED** [review]
Just one edit that I think would be important for people to consider -- it depends on the `.viam` directory where the cloud config file is located, which could be in /root/ for some people

File: README.md (hunk 1 above)
Comment (skyleilani) [reply README.md:84]: Thanks for pointing this out, I added a note about the placement of the model repository. Happy to make further changes.

**Review by bhaney — APPROVED** [review]
LGTM!

**Pull request merged**